query/QueryManager.java

```diff
@@ -50,17 +58,63 @@ public QueryManager(RPCTransaction rpcTransaction) {
 
     public Stream<ConceptMap> match(GraqlMatch query, GraknOptions options) {
         final QueryProto.Query.Req.Builder request = QueryProto.Query.Req.newBuilder().setMatchReq(
-                QueryProto.Graql.Match.Req.newBuilder().setQuery(query.toString()));
+                QueryProto.Query.Match.Req.newBuilder().setQuery(query.toString()));
         return iterateQuery(request, options, res -> res.getQueryRes().getMatchRes().getAnswersList().stream().map(ConceptMap::of));
     }
 
+    public QueryFuture<String> match(GraqlMatch.Aggregate query) {
+        return match(query, new GraknOptions());
+    }
+
+    public QueryFuture<String> match(GraqlMatch.Aggregate query, GraknOptions options) {
+        final QueryProto.Query.Req.Builder request = QueryProto.Query.Req.newBuilder().setMatchAggregateReq(
+                QueryProto.Query.MatchAggregate.Req.newBuilder().setQuery(query.toString()));
+        return runQuery(request, options, res -> res.getQueryRes().getMatchAggregateRes().getAnswer().getValue());
+    }
+
+    public Stream<AnswerGroup<ConceptMap>> match(GraqlMatch.Group query) {
+        return match(query, new GraknOptions());
+    }
+
+    public Stream<AnswerGroup<ConceptMap>> match(GraqlMatch.Group query, GraknOptions options) {
+        final QueryProto.Query.Req.Builder request = QueryProto.Query.Req.newBuilder().setMatchGroupReq(
+                QueryProto.Query.MatchGroup.Req.newBuilder().setQuery(query.toString()));
+        return iterateQuery(
+                request, options,
+                res -> res.getQueryRes().getMatchGroupRes().getAnswersList().stream()
+                        .map(e -> {
+                            Concept owner = concept(e.getOwner());
+                            List<ConceptMap> conceptMaps = e.getConceptMapsList().stream().map(ConceptMap::of).collect(Collectors.toList());
+                            return new AnswerGroup<>(owner, conceptMaps);
+                        })
+        );
+    }
+
+    public Stream<AnswerGroup<String>> match(GraqlMatch.Group.Aggregate query) {
+        return match(query, new GraknOptions());
+    }
+
+    public Stream<AnswerGroup<String>> match(GraqlMatch.Group.Aggregate query, GraknOptions options) {
+        final QueryProto.Query.Req.Builder request = QueryProto.Query.Req.newBuilder().setMatchGroupAggregateReq(
+                QueryProto.Query.MatchGroupAggregate.Req.newBuilder().setQuery(query.toString()));
+        return iterateQuery(
+                request, options,
+                res -> res.getQueryRes().getMatchGroupAggregateRes().getAnswersList().stream()
+                        .map(e -> {
```

Comment:
See above comment

Reply:
Addressed in https://github.com/graknlabs/client-java/pull/218/commits/16316dbec63658cb13c503319f3cf662ac6199d8